config/rbac/role.yaml

```diff
@@ -92,6 +92,9 @@ rules:
 - apiGroups:
   - rbac.authorization.k8s.io
   resources:
-  - '*'
+  - clusterrolebindings
   verbs:
-  - '*'
+  - get
+  - list
+  - update
+  - watch
```

Comment:
@odubajDT @james-milligan @beeme1mr what do you guys think about changing the name of this file to be a bit clearer? Maybe `manager_role.yaml`?

Reply:
If I may suggest, this is a file that is auto-generated by the kubebuilder (I didn't edit it directly, just adapted the rbac in the pod_webhook), so in case of changing the name, it may break the whole kubebuilder operator as it is relying on existence of this file and therefore would not be able to generate operator manifests. I would therefore prefer staying on the current name.